core/install_scss.js

```diff
@@ -0,0 +1,10 @@
+var Download = require('download');
```

Comment:
Haha, okay. Let me know when it's up

On Tue, Mar 17, 2015 at 9:24 AM, trevren11 <notifications@github.com> wrote:

> In core/install_scss.js
> <https://github.com/BYU-ODH/hlrdesk/pull/181#discussion_r26582459>:
>
> > @@ -0,0 +1,10 @@
> > +var Download = require('download');
>
> Ahh it left out something else too, where it installs it, slightly
> important. I will make a new branch and request. Oi
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-ODH/hlrdesk/pull/181/files#r26582459>.
>



-- 
 ian hunter